Shared/Assets.xcassets/Relay 1c 2.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "filename" : "Relay 1c 2.png",
```

Comment:
Generic comment: I hate spaces in filenames. I know it should work, but I've had so many bugs with them in the past.

Reply:
Good shout, I meant to take these out anyway. They're not used, they might as well not be adding to the size of the app.